src/types.tsx

```diff
@@ -6,9 +6,16 @@ export interface DefinitionEntry {
   pinyin: string;
   jyutping: string;
   definition: string;
+  // custom message string returned back by API
+  message: string;
 }
 
 export interface CardHeading {
   traditional: string;
   simplified?: string;
 }
+
+export const CACHE_ACTIONS = {
+  SET_CACHE: "set-cache",
+  GET_CACHE: "get-cache"
+};
```

Comment:
enum is better for this. One problem with object is that you can change the values, e.g. `CACHE_ACTIONS.SET_CACHE = "foo";`